index.html

```diff
@@ -27,6 +27,139 @@
                 <li><a href="#"><img class="social-ico" src="res/m.svg" alt="meta"></a></li>
             </ul>
-        </section>    
+        </section>
+        <section class="works-section">
+            <div class="my-recent-work">
+                <h1>My Recent Works</h1>
+                <hr class="recent-hr">
+            </div>
+            <div class="works">
+                <div class="project-1">
+                    <img class="project-1-img" src="res/img_Placeholder.png">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="siklls">
+                        <a>css</a>
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <a class="project1-btn">See Project</a>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
```

Comment:
- [ ] Kindly make sure all the cards' design sticks to the form of the Figma template e.g.: spacing between cards, See Project button width, fons size on the project detail text,... You can check all the details from Figma to match your design.

| Figma design | Your Design |
| -------------------- | ------------------- | 
| ![Screenshot from 2022-09-24 12-30-31](https://user-images.githubusercontent.com/83042742/192093377-de1c535d-b76e-40ef-826f-0279bd1f4fe5.png) | ![Screenshot from 2022-09-24 12-31-18](https://user-images.githubusercontent.com/83042742/192093390-0f960963-1e94-45be-a70f-1c1e67edc413.png) |